src/test/java/com/cydeo/runners/CucumberRunner.java

```diff
@@ -9,12 +9,13 @@
         plugin = {
                 "html:target/cucumber-reports.html",
                 "rerun:target/rerun.txt",
-                "me.jvt.cucumber.report.PrettyReports:target/cucumber"
+                "me.jvt.cucumber.report.PrettyReports:target/cucumber",
+                "json:target/cucumber.json"
         },
         features = "src/test/resources/features",
         glue = "com/cydeo/step_definitions",
         dryRun = false,
-        tags = "@B34G5-138",
```

Comment:
Line 17 is showing a conflict. I think this is because the pull request is trying to duplicate "tags". Can you try to delete this line? If this doesn't fix it then we can try something else.